common/djangoapps/django_comment_common/utils.py

```diff
@@ -1,9 +1,17 @@
 """
 Common comment client utility functions.
 """
-
 from django_comment_common.models import Role, FORUM_ROLE_ADMINISTRATOR, FORUM_ROLE_MODERATOR, \
     FORUM_ROLE_COMMUNITY_TA, FORUM_ROLE_STUDENT
+from request_cache.middleware import request_cached
+
+from .models import CourseDiscussionSettings
+from openedx.core.djangoapps.course_groups.cohorts import get_legacy_discussion_settings
+
+
+NOT_DIVIDED = CourseDiscussionSettings.NONE
```

Comment:
Not to reviewers: since everyone is going through get_course_discussion_settings/set_course_discussion_settings, it was sort of annoying to also have to import CourseDiscusssionSettings from a different file (just for NONE, COHORT, and ENROLLMENT_TRACK). Therefore I created constants in this file that point to those values.